### PR TITLE
Solve issue #1

### PR DESCRIPTION
**Pull request by deepanshu-jain1999**
Add Travis CI

### CODE REVIEW

**Review by sahitpj — COMMENTED** [review]
Hey @deepanshu-jain1999 thanks for this, looking good. 

Just  a small thing, could you add comments for each part, so that new people will understand what each refers to?

**Comment by deepanshu-jain1999** [comment]
Where I have to add comments

**Comment by deepanshu-jain1999** [comment]
Can I add .gitignore file for proper functioning of project

**Comment by sahitpj** [comment]
In the XML files and yeah sure, feel free to add a .gitignore

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# add all references that we don't want on github
+.idea/
```
Comment (sahitpj) [inline .gitignore]: Looking good! Just one more addition, just add .DS_Store in the .gitignore, I will ready to merge it

**Comment by deepanshu-jain1999** [comment]
Solved.

**Comment by sahitpj** [comment]
Thanks a lot @deepanshu-jain1999 for contributing. Would you like me to add your name to the list of contributors to PyData Gandhinagar?

**Comment by deepanshu-jain1999** [comment]
Sure.

**Pull request merged**